core/templates/components/summary-tile/subtopic-summary-tile.component.ts

```diff
@@ -75,6 +78,18 @@ export class SubtopicSummaryTileComponent implements OnInit {
     } else {
       this.thumbnailUrl = null;
     }
+    this.subtopicTitleTranslationKey = this.i18nLanguageCodeService.
+      getSubtopicTranslationKey(
+        this.topicId, this.subtopic.getUrlFragment(),
+        TranslationKeyType.TITLE);
+  }
+
+  isHackySubtopicTitleTranslationDisplayed(): boolean {
+    return (
+      this.i18nLanguageCodeService.isHackyTranslationAvailable(
+        this.subtopicTitleTranslationKey) &&
+        !this.i18nLanguageCodeService.isCurrentLanguageEnglish()
```

Comment:
Deindent this by 2, since it's in the outer scope. Or, break the previous line before the ')' and put the ') &&' at the beginning of this line (again deindented by 2).

Reply:
Done!